lib/constants.dart

```diff
@@ -44,4 +44,8 @@ class Constants {
   static const String UNAUTHENTICATED =
       'Unauthenticated to perform this action';
   static const String UNAUTHORIZED = 'Unauthorized to perform this action';
+
+  ////Country-Institute Constants
+  static const String COUNTRY = 'country';
+  static const String EDUCATIONAL_INSTITUTE = 'educational institute';
```

Comment:
These constants have the scope only for its API model so it would be better if they are in its API model.

Reply:
Currently, I haven't created a separate class for API model and hence I have included them in the Constants itself. Should that work?.

 However , if you want me to create a model for these two APIs I can do that. All that the class is going to have is two constants.

Reply:
I meant to keep them in its API class.